sdk/eventhub/Azure.Messaging.EventHubs/CONTRIBUTING.md

```diff
@@ -1,19 +1,70 @@
+ Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use any contribution that you make. For details, visit https://cla.microsoft.com.
 
-### Live test information
+## Code of conduct
 
-### Target frameworks
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-For information about the target frameworks of the Azure Event Hubs client library, please refer to the [Target Frameworks](https://github.com/azure/azure-sdk-for-net#target-frameworks) of the Microsoft Azure SDK for .NET.  
+## Getting started
 
-### Versioning information
+Before working on a contribution, it would be beneficial to familiarize yourself with the process and guidelines used for the Azure SDKs so that your submission is consistent with the project standards and is ready to be accepted with fewer changes requested.   In particular, it is recommended to review:
 
-The Azure Event Hubs client library uses [the semantic versioning scheme.](http://semver.org/)
+  - [Azure SDK README](https://github.com/Azure/azure-sdk), to learn more about the overall project and processes used.
+  - [Azure SDK Design Guidelines](https://azuresdkspecs.z5.web.core.windows.net/DesignGuidelines.html#general-documentation), to understand the general guidelines for the SDKs across all languages and platforms
+  - [Azure SDK Design Guidelines for .NET](https://azuresdkspecs.z5.web.core.windows.net/DotNetSpec.html), to understand the guidelines specific to the Azure SDKs for .NET.
 
-### Code of conduct
+## Development environment setup
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+### Running tests
+
+The Event Hubs client library tests may be executed using the `dotnet` CLI, or the test runner of your choice - such as Visual Studio or Visual Studio Code.  For those developers using Visual Studio, it is safe to use the Live Unit Testing feature, as any tests with external dependencies have been marked to be excluded.
+
+Tests in the Event Hubs client library are split into two categories - unit tests and integration tests.  For unit tests, there are no special considerations; these are self-contained and execute locally without any reliance on external resources.  Unit tests are considered the default test type in the Event Hubs client library and, thus, have no explicit category trait attached to them.  
```

Comment:
I wonder if this would look nicer as having bullets, or sections for the two test categories, rather than trailing the opening sentence with unit tests and a new paragraph for integration tests?

Reply:
Reformatted;  the second bullet is a bit chunky, but I think that I like the flow better.